pastehunter/outputs/http_output.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import requests
+import json
+from common import parse_config
+
+logger = logging.getLogger('pastehunter')
+
+config = parse_config()
+
+class HttpOutput():
+    def __init__(self):
+        self.valid = True
+        self.endpoint_url = config['outputs']['http_output']['endpoint_url']
+        self.http_auth = config['outputs']['http_output']['http_auth']
+        self.http_user = config['outputs']['http_output']['http_user']
+        self.http_password = config['outputs']['http_output']['http_password']
+
+        if self.endpoint_url == '':
+            logging.error("HTTP endpoint not configured")
+            self.valid = False
+
+    def store_paste(self, paste_data):
+        if self.valid:
+            
+            json_data = paste_data
+            del json_data['raw_paste']
+            json_data['@timestamp'] += 'Z'
```

Comment:
I am guessing this is for a specific use case? I don't think adding the Z is generally applicable for this.

Reply:
It is to respect the format of ISO 8601, I had set GMT by default, but I have just added a setting to define the timezone